wasm-test/src/lib.rs

```diff
@@ -53,7 +54,9 @@ pub extern "C" fn check_commit_entry(encoded_allocation_of_input: u32) -> u32 {
     }
     let input: CommitInputStruct = result.unwrap();
 
-    let res = hdk::commit_entry(&input.entry_type_name, &input.entry_content);
+    let res = hdk::commit_entry(&input.entry_type_name, json!({
+        "entry_content": &input.entry_content
```

Comment:
why is it a JSON object that has 'entry_content' as a key?

Reply:
Because whatever. It's irrelevant for this test. I just had to change this line to provide a second parameter that is a `serde_json::Value`.

Reply:
Ok cool. Gotcha